components/learn/modules/ROOT/pages/unit-testing.adoc

```diff
@@ -171,14 +170,6 @@ describe('Box', function () {
     // Test that a ValueChanged event was emitted with the new value
     expectEvent(receipt, 'ValueChanged', { newValue: value });
   });
-
```

Comment:
`Box` _is_ `Ownable` [here](https://full-site--openzeppelin-docs.netlify.com/learn/writing-smart-contracts#box-contract). Did the tests fail on your end?

Reply:
Whilst Box is made Ownable  in writing-smart-contracts in the previous document deploying-and-interacting.adoc https://full-site--openzeppelin-docs.netlify.com/learn/deploying-and-interacting#deploying-a-smart-contract (if following in order) it is not Ownable so the test fails.

So we either update the deploying-and-interacting to make that version of Box Ownable or remove the failing test.

For now I removed the failing test

Reply:
The issue is I wanted that test to showcase `expectRevert` behavior :/ I'll try to tackle that via prose

Reply:
Pesky, as it is nice to have the test.